docs/source/joblib.rst

```diff
@@ -52,6 +52,15 @@ validated parameter search as follows.
    with parallel_backend('dask.distributed', scheduler_host='localhost:8786'):
        search.fit(digits.data, digits.target)
 
+## Nested Parallelism
+
+Note that this code will not work if you distribute it via dask in order to implement nested parallelism. The reason is that each worker will have a fresh copy of sklearn and `from sklearn.externals.joblib import parallel_backend` will not apply to them, resulting in a "KeyError" when trying to distribute, but no problem when running the inner loop locally. The solution is 
+
+.. code-block:: python
+
+    from distributed.joblib import DistributedBackend 
+    register_parallel_backend('distributed', DistributedBackend, make_default=True)
```

Comment:
And I can't do any waiting on futures in this code because none of the parts are dependent on each other - it's 8 independent runs, each of which contains 2 independent folds of crossvalidation.